### PR TITLE
Changed bsg_newlib to dramfs

### PR DESCRIPTION
**Pull request by vb000**
Changed bsg_newlib name to dramfs. Reflecting that change here.

### CODE REVIEW

File: software/spmd/common/spike.ld [hunk 1]
```diff
@@ -33,7 +33,7 @@ SECTIONS
 
   .text : 
   {
-     *crt.o(.text)
+     *(.crtbegin)
```
Comment (tommydcjung) [inline software/spmd/common/spike.ld]: maybe add a comment why this change was necessary?

Comment (vb000) [reply software/spmd/common/spike.ld]: done!
